zerver/lib/message.py

```diff
@@ -655,7 +655,10 @@ def build_dict_from_raw_db_row(row: RawReactionRow) -> Dict[str, Any]:
 
 
 def access_message(
-    user_profile: UserProfile, message_id: int
+    user_profile: UserProfile,
+    message_id: int,
+    lock_message: bool = False,
+    lock_usermessage: bool = False,
```

Comment:
These probably deserve a bit of documentation in the docstring about when to use them.

Reply:
Done.